File: auth_samples_flutter/lib/auth_config.dart
class AuthConfig {
  static const authorizationEndpoint = 'https://test.id.tuurio.com/oauth2/authorize';
  static const tokenEndpoint = 'https://test.id.tuurio.com/oauth2/token';
  static const issuer = 'https://test.id.tuurio.com';
  static const clientId = 'spa-K53I';
  static const redirectUri = 'com.example.app://oauth2redirect';
  static const postLogoutRedirectUri = 'http://localhost:5173/';
  static const scopes = ['openid', 'profile', 'email'];
}

File: auth_samples_android/app/src/main/java/com/tuurio/authsample/auth/AuthConfig.kt
package com.tuurio.authsample.auth

import android.net.Uri

object AuthConfig {
  val authorizeEndpoint: Uri = Uri.parse("https://test.id.tuurio.com/oauth2/authorize")
  val tokenEndpoint: Uri = Uri.parse("https://test.id.tuurio.com/oauth2/token")
  const val clientId: String = "spa-K53I"
  val redirectUri: Uri = Uri.parse("com.example.app://oauth2redirect")
  const val scope: String = "openid profile email"

  val discoveryUri: Uri = Uri.parse("https://test.id.tuurio.com/.well-known/openid-configuration")
  val postLogoutRedirectUri: Uri = Uri.parse("http://localhost:5173/")
}
